domains/next.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Zero-Ejs",
+        "email": "ramkrishnadeveloper1@gmail.com"
+    },
+    "record": {
+        "URL": "next.is-a.dev"
```

Comment:
- URL record must have the protocol (`https://`)
- you can't redirect the website to itself, you'd make a redirect loop